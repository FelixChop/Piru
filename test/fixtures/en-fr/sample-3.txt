After the storm, the streets were littered with detritus, but the neighborhood pulled together to clean up the mess in short order.
