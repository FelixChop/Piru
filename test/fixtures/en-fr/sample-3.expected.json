[
  {
    "word": "littered",
    "lemma": "to litter",
    "definition": "couvrir, joncher",
    "citation": "After the storm, the streets were littered with detritus, but the neighborhood pulled together to clean up the mess in short order.",
    "to_learn": true
  },
  {
    "word": "pulled together",
    "lemma": "to pull together",
    "definition": "unir ses efforts",
    "citation": "After the storm, the streets were littered with detritus, but the neighborhood pulled together to clean up the mess in short order.",
    "to_learn": true
  },
  {
    "word": "in short order",
    "lemma": "in short order",
    "definition": "rapidement",
    "citation": "After the storm, the streets were littered with detritus, but the neighborhood pulled together to clean up the mess in short order.",
    "to_learn": true
  }
]
